pyfakefs/tests/fake_pathlib_test.py

```diff
@@ -632,6 +632,7 @@ def test_symlink_to(self):
         self.assertTrue(path.is_symlink())
 
     @unittest.skipIf(sys.version_info < (3, 8), "link_to new in Python 3.8")
+    @unittest.skipIf(sys.version_info >= (3, 12), "link_to removed in Python 3.12")
```

Comment:
https://github.com/python/cpython/blob/0397f040e239c82848f51dedef9af7b9bd99ef55/Lib/pathlib.py#L1222-L1225

Reply:
Yes, this had been deprecated before (actually renamed to `hardlink_to`).